test/Swashbuckle.AspNetCore.IntegrationTests/SwaggerIntegrationTests.cs

```diff
@@ -21,6 +21,8 @@ public class SwaggerIntegrationTests
         [Theory]
         [InlineData(typeof(Basic.Startup), "/swagger/v1/swagger.json")]
         [InlineData(typeof(CliExample.Startup), "/swagger/v1/swagger_net8.0.json")]
+        // TODO: FIXME: CliExampleWithFactory project is not working
+        //[InlineData(typeof(CliExampleWithFactory.Startup), "/swagger/v1/swagger_net8.0.json")]
```

Comment:
I'd rather these TODOs are fixed in the same PR _or_ a separate PR is merged first that fixes them and removes the need for the TODOs.

Reply:
Fixed